modules/administration-guide/examples/proc_che-rolling-the-upgrade-back-from-che-7-49-to-che-7-41-on-openshift.adoc

```diff
@@ -0,0 +1,46 @@
+:_content-type: PROCEDURE
+
+:parent-context-of-rolling-the-upgrade-back-to-che-7.41-on-red-hat-openshift: {context}
+
+[id="rolling-the-upgrade-back-to-che-7.41-on-red-hat-openshift_{context}"]
```

Comment:
Why didn't I notice this before? :( Please, unify the title and the ID here, too.